docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+---
+title: Teleport integration with Slack
+description: This guide explains how to set up a Slack plugin for Teleport for privilege elevation approvals.
+h1: Teleport Slack Plugin Setup
+---
+
+This guide examines how to configure Teleport for use with Slack.
+
+Slack integration allows you to treat Teleport access and permission requests within Slack messages as well as through the command line.
+
+<Admonition type="Note" title="Access Requests">
+  Access Requests are now supported in both the Teleport Open Source and Enterprise Editions.
+</Admonition>
+
+#### Example Slack request
+
+<video controls>
+  <source src="../../../img/enterprise/plugins/slack/slack.mp4" type="video/mp4" />
```

Comment:
The image links should point to goteleport?

Reply:
Yes, surprisingly the video lives in the `img` dir. Happy to relocate that (may not be best from a CDN or serving standpoint).